File: 프로그래머스/0/120822. 문자열 뒤집기/문자열 뒤집기.cs
using System;
using System.Collections.Generic;

public class Solution 
{
    public string solution(string my_string) 
    {
        List<char> rvStr = new List<char>(); 

        for(int i=my_string.Length-1; i>=0; i--)
        {
            rvStr.Add(my_string[i]);
        }
        
        char[] rvArr = rvStr.ToArray();
        string reverse = new string(rvArr);
        return reverse;
    }
}

File: 프로그래머스/0/120822. 문자열 뒤집기/문자열 뒤집기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string my_string) 
{
    string result = "";
    for(int i=my_string.size()-1; i>=0; i--)
    {
        result += my_string[i];
    }
    return result;
}